app/lib/efile/id/id39_r_calculator.rb

```diff
@@ -85,8 +85,8 @@ def calculate_sec_b_line_8d
       end
 
       def calculate_sec_b_line_8e
-        @intake.state_file1099_rs.sum do |form1099r|
-          if form1099r.state_specific_followup&.eligible_income_source_yes? && form1099r.taxable_amount.present?
```

Comment:
taxable_amount presence is checked in `eligible_1099rs`, so it was a redundant check